### PR TITLE
add FS function

### PR DESCRIPTION
**Pull request by pkdotson**

### CODE REVIEW

File: Fs.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+
+const names = (name) => {
+  if(name != 'Phillip'){
+    console.log(name);
```
Comment (songz) [inline Fs.js]: can you write this into a file in your directory?

**Review by seemcat — APPROVED** [review]
Try making it shorter next time! But this works.

File: prntname.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+READMEalopezameybettybharathbillcecilecolindsoperemilyemmanuelgitgitlab-runnerhuejiajiarunkinuelouismadhumadhubmanikmcmikemingoliviapdiddypeterahulsahilsongsyatriyj
```
Comment (SahilKalra98) [inline prntname.txt]: Don't commit generated files :)

File: Fs.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+
+let s = '';
+
+const names = (name) => {
+  if(name != 'Phillip'){
+  s = s + name;
```
Comment (songz) [inline Fs.js]: what is `s`? better variable names please.

File: Fs.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+
+let s = '';
```
Comment (allopez7) [inline Fs.js]: use a more descriptive name for s

File: Fs.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+const names = (name) => {
+  if(name != 'Phillip'){
+  string = string + name; 
+  }
+}
+
+const allNames = (err, files) => {
+  files.forEach(names);
+  fs.writeFile('prntname.txt', s, (err) => {
```
Comment (allopez7) [inline Fs.js]: The s was not changed to string here.

**Review by allopez7 — CHANGES_REQUESTED** [review]
One change required.

File: Fs.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+
+let string = '';
```
Comment (songz) [inline Fs.js]: string is a really bad variable name. Let's call it `allNamesWithoutMe`

File: Fs.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
```
Comment (songz) [inline Fs.js]: Your file name is `Fs.js`. Please don't capitalize your file name.

File: fs.js [hunk 8]
```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+
+let allNamesWithoutMe = '';
+
+const names = (name) => {
+  if(name != 'Phillip'){
+  allNamesWithoutMe = allNamesWithoutMe + ' ' + name;
```
Comment (songz) [inline fs.js]: add 2 space!

**Pull request merged**